conf/modules.config

```diff
@@ -589,6 +589,40 @@ process {
         ]
     }
 
+    //
+    //  BEDTOOLS_COVERAGE
+    //
+    withName: SAMTOOLS_VIEW_GENOME {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.args = '-H --output-fmt sam'
+        ext.prefix = "genome"
+        ext.args2 = "&& cat genome.sam |  grep '@SQ' | sed 's#@SQ\tSN:\\|LN:##g' > genome.txt && mv genome.txt genome.sam"
+        publishDir = [
+            enabled: false
+        ]
+    }
+    withName: BEDTOOLS_COVERAGE_MEAN {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.args = '-mean -nonamecheck'
```

Comment:
```suggestion
        ext.args = '-mean -nonamecheck -sorted'
```

Reply:
We won't need the -sorted here because it is added when you supply the genome.txt file.

Reply:
Ahh ok!